### ISSUE TITLE
`run_procedure()` unconditionally renders `subdataset()` output

### ISSUE THREAD

**Issue by mih** [datalad/datalad#7091]
Discovered in https://github.com/datalad/datalad-gooey/issues/373

Fix

```diff
diff --git a/datalad/local/run_procedure.py b/datalad/local/run_procedure.py
index 4d195b1de..72e7ee0ae 100644
--- a/datalad/local/run_procedure.py
+++ b/datalad/local/run_procedure.py
@@ -141,7 +141,8 @@ def _get_procedure_implementation(name='*', ds=None):
                 yield (m, n,) + _get_proc_config(n, ds=ds)
         # 2.1. check subdatasets recursively
         for subds in ds.subdatasets(return_type='generator',
-                                    result_xfm='datasets'):
+                                    result_xfm='datasets',
+                                    result_renderer='disabled'):
             for m, n, f, h in _get_procedure_implementation(name=name, ds=subds):
                 yield m, n, f, h
 
```

**Comment by yarikoptic-gitmate** [comment]
Issue fixed in [`0.17.8`](https://github.com/datalad/datalad/releases/tag/0.17.8)

**Comment by christian-monch** [comment]
Fixed by PR #7094
